### PR TITLE
design update to new zeplin design

### PR DESCRIPTION
**Pull request by mritunjayz**

### CODE REVIEW

**Comment by mritunjayz** [comment]
WIP

**Comment by mritunjayz** [comment]
#2189 #2204

**Comment by mritunjayz** [comment]
#2203

File: web/src/components/pages/profile/avatar-setup/avatar-setup.tsx [hunk 1]
```diff
@@ -270,32 +274,60 @@ class AvatarSetup extends React.Component<Props, State> {
...
-      this.setState({ counter: tl });
-      if (this.state.counter <= 0) {
-        this.setState({ clipStatus: 'started' });
-        const { locale } = this.props;
-        trackVoiceAvatar('create-voice-avatar', locale);
-        this.handleRecordClick();
-        clearInterval(downloadTimer);
-      }
-    }, 1000);
+    if (this.state.clipStatus === 'notStarted') {
```
Comment (Gregoor) [inline web/src/components/pages/profile/avatar-setup/avatar-setup.tsx]: I'd recommend doing a early return here for the else case instead, i.e. `if (this.state.clipStatus !== 'notStarted') return`. That would reduce nesting which imo makes for more readable code

**Comment by mritunjayz** [comment]
I had thought of it to not combine multiple works in it but I do not know why I did this :confused:  . Next time I will make specific PR to issues.

**Comment by Gregoor** [comment]
Great, thanks :+1:

**Pull request merged**